### PR TITLE
Add espduino boards

### PR DESCRIPTION
**Pull request by SmartArduino**
ESPDuino boards from doit.am (arduino uno look like board for esp8266)

### CODE REVIEW

**Comment by Links2004** [comment]
please combine #1305, #1304, #1303, #1302 in one pull request, thanks.

File: boards.txt [hunk 1]
```diff
@@ -6,7 +6,73 @@ menu.FlashFreq=Flash Frequency
 menu.UploadTool=Upload Using
 menu.ResetMethod=Reset Method
 menu.ESPModule=Module
-
+##############################################################
+espduino.name=ESPDuino (ESP-13 Module)
```
Comment (me-no-dev) [inline boards.txt]: should this board go on top? I think it needs to go below the generic board

Comment (SmartArduino) [inline boards.txt]: Yes you are right.

**Comment by SmartArduino** [comment]
I'm very sorry but I do not know how to combine #1305, #1304, #1303, #1302 in one pull request. I edited all the scirpts or files in a browser online. 
A github client must be used?

**Pull request closed without merging**